src/App.js

```diff
@@ -1,12 +1,14 @@
 import React from 'react'
 import logo from './assets/logo192.png'
+import ExpenseForm from './components/ExpenseForm/ExpenseForm'
 
 function App() {
   return (
     <div>
       <header style={{ textAlign: 'center', marginTop: 20 }}>
```

Comment:
why this is not in styled component pattern?

Reply:
This was done before by Gadzet